intro/utils.py

```diff
@@ -0,0 +1,27 @@
+import glob
+import os
+import subprocess
+
+import pandas as pd
+
+
+def load_unlabeled_spam_dataset():
+    """Load spam training dataset without any labels."""
+    if os.path.basename(os.getcwd()) == "snorkel-tutorials":
+        os.chdir("intro")
+    subprocess.call("bash download_data.sh", shell=True)
```

Comment:
align with spam dir one. for instance, should be `subprocess.run([...], check=True)`